treeherder/log_parser/tasks.py

```diff
@@ -57,5 +58,8 @@ def store_error_summary(project, job_log_url, job_guid):
     try:
         logger.info('Running store_error_summary')
```

Comment:
Actually ignore that, it's only the celery worker INFO lines that I can see, the rest are error-only afaict.